src/modules/CanvasResize/Module.js

```diff
@@ -1,6 +1,8 @@
+const ndarray = require('ndarray');
```

Comment:
ndarray is imported here

Reply:
Are you saying it shouldn't be necessary? Just checking, let's make a code suggestion perhaps?

Reply:
ah i see it's required again below. I'll remove that one.